src/components/dashboard/Dashboard.js

```diff
@@ -396,7 +394,7 @@ const Dashboard = props => {
 
     // setTimeout(animateItems, marketAnimationDuration)
 
-    initTransferEvents(gdstore)
+    await initTransferEvents()
```

Comment:
why do we need to await?

Reply:
@sirpy 
because it's async now
it awaits for .ready inside

Reply:
yes but we want it to happen in background and not block loading the dashboard

Reply:
the await for initTransferEvents is irrelevant for the await inside it for .ready
it only means that any  initTransferEvents will not block the rest of the init function it is called from